doc/supportedPlatformVersions.md

```diff
@@ -757,6 +763,7 @@
 - 7.4.26
 - 7.4.28
 - 7.4.30
+- 8.0.22
 - 8.1.0
```

Comment:
Do we need these platform versions to be added here?

Reply:
auto generated..

Reply:
Yeah looks like whenever php 8.0.22 was added for bullseye, the generateConstants script wasn't run. So now that it has run, we are seeing it resolve all the discrepancies from the versionsToBuild.txt files and this file